src/components/LogViewer/History.tsx

```diff
@@ -41,6 +41,38 @@ const formatTimestamp = (timestamp: number) => {
     .join(':');
 };
 
+const getFilterTag = (appendToQuery: Function, log: LogEntry) => {
```

Comment:
getFilterTag is really a functional component, how would you feel about:

```js
const FilterTag: React.FC<{...}> = ({appendToQuery, log}) => {
  return <div>...</div>
};
```

and 

```jsx
<FilterTag log={log} appendToQuery={appendToQuery} />
```

Reply:
smart smart